src/main/java/org/ggp/base/apps/server/scheduling/Scheduler.java

```diff
@@ -65,7 +65,7 @@ public synchronized void addPendingMatch(PendingMatch spec) {
 
     private synchronized boolean canSchedule(PendingMatch spec) {
         for (PlayerPresence player : spec.thePlayers) {
-            if (!player.getStatus().equals("available")) {
+            if (!player.getStatus().toLowerCase().equals("available")) {
```

Comment:
A better alternative is equalsIgnoreCase, but this is good enough to get this fixed.